### ISSUE TITLE
Support for / ignore comma separated KUBECONFIG env paths

### ISSUE THREAD

**Issue by alexdyas** [danielfoehrKn/kubeswitch#24]
Hello,

The Kubernetes documentation states that a colon separated list of paths is supported for the KUBECONFIG environment variable :

https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#linux

However kubeswitch seems to have trouble with this : 

```
% echo $KUBECONFIG
/home/alex/.kube/configs/first.config:/home/alex/.kube/configs/second.config
% switch
Error: the configured kubeconfig directory "/home/alex/.kube/configs/first.config:/home/alex/.kube/configs/second.config" does not exist
the configured kubeconfig directory "/home/alex/.kube/configs/first.config:/home/alex/.kube/configs/second.config" does not exist
```

Am I doing something wrong?

Cheers,

Alexander

**Comment by danielfoehrKn** [comment]
I created [this commit](https://github.com/danielfoehrKn/kubeswitch/commit/8b537560ac84b2e603c9f2fe5e3aeff35ca47948) included in the release `0.4.8` so you can try it out.

**Comment by alexdyas** [comment]
Hi, sorry I missed this being closed. Just tried it, works perfectly, thanks!
